### PR TITLE
feat: Empty Flutter Package

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [x] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [x] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## What is the current behavior?

N/A.

## What is the new behavior?

Empty Flutter Package.

## Checklist

Please check that your PR fulfills the following requirements:

- [x] Documentation has been added/updated.
- [x] Automated tests for the changes have been added/updated.
- [x] Updated [CHANGELOG.md](../CHANGELOG.md) (if you introduced a breaking change).

### CODE REVIEW

**Review by Marc-Antoine-Soucy — APPROVED** [review]

**Review by takla21 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Please check that your PR fulfills the following requirements:
 
 - [ ] Documentation has been added/updated.
 - [ ] Automated tests for the changes have been added/updated.
-- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).
+- [ ] Updated [CHANGELOG.md](../CHANGELOG.md) (if you introduced a breaking change).
```
Comment (jeanplevesque) [inline .github/PULL_REQUEST_TEMPLATE.md]: Remove `(if you introduced a breaking change)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# 0.1.0+1
```
Comment (jeanplevesque) [inline CHANGELOG.md]: Not sure this version is relevant

File: pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: review_service
+description: A multi-platform recipe to rate mobile apps.
+version: 0.1.0+1
+homepage: https://github.com/nventive/ReviewService-Dart
+
+environment:
+  sdk: "^3.3.0"
+  flutter: "^3.19.0"
```
Comment (jeanplevesque) [inline pubspec.yaml:8]: Do we need Flutter or could it be Dart-only?

Comment (Soap-141) [reply pubspec.yaml:8]: Flutter, because of the prompter.

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**